### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by Psychedelic2** [martinet101/ElevenClock#255] [General]
The clock problem has been bothering me for a month, but only now I decided to google it. Good job, thank you, works on all 4 monitors.

**Comment by FaceCutter** [comment]
Thank you so much!

**Comment by jonnyka** [comment]
@martinet101 I just wanna thank you, you're freaking awesome - I've asked for the ability to change the font size, and you updated it in days
thanks again man!
